xdsl/ir/core.py

```diff
@@ -1406,6 +1434,12 @@ def predecessors(self) -> tuple[Block, ...]:
         return tuple(
             p for use in self.uses if (p := use.operation.parent_block()) is not None
         )
+    
+    def successors(self) -> tuple[Block, ...]:
+        terminator = list(self.ops)[-1]
+        return tuple(
+            successor for successor in terminator.successors
+        )
```

Comment:
does this deserve a helper? If so, I would strongly recommend against creating a list only to get its last element, `self.last_op` will do the same avoiding the O(n) list creation

Reply:
Also better in own PR with small unit test